micrometer/src/main/java/feign/micrometer/MeteredClient.java

```diff
@@ -13,15 +13,15 @@
  */
 package feign.micrometer;
 
-import java.io.IOException;
-import feign.Client;
-import feign.FeignException;
-import feign.Request;
+import feign.*;
 import feign.Request.Options;
-import feign.RequestTemplate;
-import feign.Response;
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.Timer;
+import java.io.IOException;
```

Comment:
it would be nice not to change import ordering... but, don't worry too much about it =)